docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -13,16 +13,89 @@ weight: 850
+1. On the **Panel editor**, enter a **Title** for your trace panel.
+1. Add an appropriate TraceQL query to search for traces that you would like to visualize in the dashboard. We’ll use a simple, static query as an example. You can write the TraceQL query as a template query to take advantage of other dashboard variables, if they exist. This lets you create dynamic queries based on these variables. <br /> {{< figure src="/static/img/docs/panels/traces/screenshot-traces-dynamic-query.png" caption="Create a dynamic query" >}}
+
+When results are returned from a query, the results are rendered in the panel’s table.
+
+{{< figure src="/static/img/docs/panels/traces/screenshot-traces-returned-query.png" caption="Results of a returned query in the panel table" >}}
+
+### Use a variable to add other links to traces
+
+Traces results include links to an Explorer page that render the trace. You can add other links to traces in the table that fill in the `traceId` dashboard variable when selected so that the trace is visualized in the same dashboard.
+
+To do so, we’re going to create a set of data links in the panel.
+
+1. Select **Add link** from the **Data links** section of the right menu.
+1. Add a **Title** for the data link.
+1. Find the UUID of the dashboard by looking in your browser’s address bar when the full dashboard is being rendered. Because this is a link to a dashboard in the same Grafana instance, only the path of the dashboard is required. <br /> {{< figure src="/static/img/docs/panels/traces/screensnot-traces-uuid-url.png" caption="Unique identifier for the dashboard" >}}
+
+1. In the `URL` edit field, we are going to make self-reference to the dashboard that contains both of the panels. This self-reference uses the value of the selected trace in the table to fill in the dashboard variable. We’ll use the path for the dashboard from the previous step and then fill in the value of `traceId` using the selected results from the TraceQL table. The trace ID is exposed using the `traceID` data field in the returned results, so we use that as the value for our dashboard variable. <br/> {{< figure class="float-right"  src="/static/img/docs/panels/traces/screenshot-traces-edit-link.png" caption="Edit link and add the Trace link" >}}
+
+1. Select **Save** to save the data link.
+1. Select **Apply** from the panel editor to apply the panel to the dashboard.
+   Extend the width of the new table panel so that it covers the width of your dashboard.
+   It’s also worth saving your dashboard at this point!
+
+You should now see a list of matching traces in the Table visualization. While selecting the `TraceID` or `SpanID` fields will give you the option to either open an Explorer page to visualize the trace or following the data link, selecting any other field (such as `Start time`, `Name` or `Duration`) automatically follows the data link, filling in the `traceId` dashboard variable, and then shows the relevant trace in the trace panel.
```

Comment:
Since these are now fields in the UI, giving them the same UI element formatting.
```suggestion
You should now see a list of matching traces in the Table visualization. While selecting the **TraceID** or **SpanID** fields will give you the option to either open the **Explore** page to visualize the trace or following the data link, selecting any other field (such as **Start time**, **Name** or **Duration**) automatically follows the data link, filling in the `traceId` dashboard variable, and then shows the relevant trace in the trace panel.
```

Reply:
Thank you.